apiary.apib

```diff
@@ -3,245 +3,212 @@ HOST: http://lore.mit.edu/
 
 # LORE
 
-Simple API on top of MIT LORE application
+A REST API on top of MIT LORE application.
```

Comment:
That's a good idea. Do you suggest using this text or should I write something myself?